src/main/java/org/apache/commons/lang3/RandomStringUtils.java

```diff
@@ -34,7 +34,11 @@
  * RandomStringGenerator</a> instead.</p>
  *
  * <p>Caveat: Instances of {@link Random}, upon which the implementation of this
- * class relies, are not cryptographically secure.</p>
+ * class relies, are <b>not cryptographically secure</b>.
+ * Do not use this classes' default implementation of {@link Random} in security sensitive locations,
+ * for example password reset key generation, as all future values can be computed as proven by
+ * <a href="https://medium.com/@alex91ar/the-java-soothsayer-a-practical-application-for-insecure-randomness-c67b0cd148cd">
```

Comment:
Do we really want to link to a site that only let's you read a limited number of articles per month? Isn't there a freer source for this information?

Reply:
As an author that regularly publishes on medium, you can choose to monizie or not monizie. It doesn't look like this one is monizied. I can ask the author for a "friend" link that doesn't have that pop-up when you load it though.

Reply:
```suggestion
 * <a href="https://medium.com/@alex91ar/the-java-soothsayer-a-practical-application-for-insecure-randomness-c67b0cd148cd?source=friends_link&sk=3db1c41cc81a58f70ed05a7315191385">
```

Reply:
I say we KISS it and put in the comment without a link.

Reply:
I mean, it's not that hard to find if you know what to Google. But this makes a software engineer/PEN Tester/security researchers life easier by fully explaining the potential impacts of their choices here. I figure the more documentation, the better.